find_number.rb

```diff
@@ -0,0 +1,26 @@
+secret_number = rand(1..100)
+
+puts 'Guess the number'
+number_guess = 0
+
+while true
+    puts "Enter number from 1 to 100"
+    number = gets.to_i
+    number_guess += 1
+    if number == secret_number
+        puts "You guessed the number #{secret_number} in #{number_guess} try/tries."
+        puts "Do you want to play again? Print Y or N"
+        if gets.chomp.upcase == "Y"
+            secret_number = rand(1..100)
+            number_steps = 0
```

Comment:
`number_steps` isn't referenced anywhere else, so I think that this is a bug and that you instead want:

```suggestion
            number_guess = 0
```

Reply:
done